src/rules/order.js

```diff
@@ -243,28 +247,22 @@ function makeOutOfOrderReport(context, imported) {
   reportOutOfOrder(context, imported, outOfOrder, 'before')
 }
 
-function importsSorterAsc(importA, importB) {
-  if (importA < importB) {
-    return -1
-  }
-
-  if (importA > importB) {
-    return 1
-  }
-
-  return 0
-}
+function getSorter(ascending) {
+  return function importsSorter(importA, importB) {
+    let result
 
-function importsSorterDesc(importA, importB) {
-  if (importA < importB) {
-    return 1
-  }
+    if ((importA < importB) || importB === null) {
+      result = -1
+    }
+    else if ((importA > importB) || importA === null) {
+      result = 1
+    }
+    else {
+      result = 0
+    }
 
-  if (importA > importB) {
-    return -1
+    return result * (ascending ? 1 : -1)
   }
-
-  return 0
 }
```

Comment:
can you elaborate on why you prefer this approach, to the current one of a separate function?

Reply:
Because both, the asc- and the desc-sorter looked very identical to me I thought it'd simplify the code when merging both functions into one.

Reply:
the intention was to avoid the `ascending` check every time - ie, for performance.

Reply:
Aaaah got it! 😄 
No worries then.
The `ascending` check is done only once here, too:
https://github.com/manuth/eslint-plugin-import/blob/2c61a5290e3211c8ea8127d9d1326e109d885080/src/rules/order.js#L279

Reply:
So I just tried to decrease the calls of the conditional `ascending` operation.
Does it work out that way?